frontend/src/hooks/useConfig.ts

```diff
@@ -0,0 +1,11 @@
+import { ref, computed } from 'vue';
+
+export const useConfig = () => {
+  const isLocalNetwork = ref(window.location.hostname === 'localhost');
```

Comment:
@denishacquin I would decide this based on an env variable

Reply:
That was actually my first intention, I just decided to go a for a quicker solution. But you're absolutely correct. And thanks for the copying tip, wasn't aware of that.